README.md

```diff
@@ -11,25 +11,91 @@ _docs:_
 
 ## Motivation
 
-Clojure's built-in data structures get you a long way when trying to codify and solve difficult problems. A solid selection of core functions allow you to easily create and access core data structures. Unfortunately, this flexibility does not extend to testing: a comprehensive yet extensible way to assert that the data fits a particular structure seems to be lacking.
+Clojure's built-in data structures get you a long way when trying to codify and solve difficult problems. A solid selection of core functions allow you to easily create and access core data structures. Unfortunately, this flexibility does not extend to testing: we seem to be missinng a comprehensive yet extensible way to assert that the data fits a particular structure.
 
-This library addresses this issue by providing composable matcher combinators that can be used as building blocks to effectively test functions that evaluate to nested data-structures.
+This library addresses this issue by providing composable matcher combinators that can be used as building blocks to test functions that evaluate to nested data-structures more effectively.
 
 ## Features
 
+- Matchers for scalar and structural values
+  - Good readability supported by default interpretations of Clojure types as matchers
 - Pretty-printed diffs when the actual result doesn't match the expected matcher
-- Integrates with `clojure.test` and `midje`
-- Good readability by providing default interpretations of core Clojure data-structures as matcher combinators
+- Integration with `clojure.test` and `midje`
 
-| Midje checkers | Matcher combinators |
-| ------- | ----- |
-| ![midje checkers](doc/images/midje_check.png) | ![matcher combinators check](doc/images/matcher_check.png) |
+## Usage
 
-| Midje checkers failure output | Matcher combinators failure output |
-| ------- | ----- |
-| ![midje checker failure output](doc/images/midje_failure.png) | ![matcher combinators failure output](doc/images/matcher_output.png) |
+### `clojure.test`
 
-## Usage
+Require the `matcher-combinators.test` namespace, which will extend `clojure.test`'s `is` macro to accept the `match?` and `thrown-match?` directives.
+
+ - `match?`: The first argument should be the matcher-combinator represented the expected value, and the second argument should be the expression being checked.
+ - `thrown-match?`: The first argument should be a throwable subclass, the second a matcher-combinators, and the third the expression being checked.
+
+For example:
+
+```clojure
+(require '[clojure.test :refer [deftest is]]
+         '[matcher-combinators.test] ;; adds support for `match?` and `thrown-match?` in `is` expressions
```

Comment:
oh, so this is the way it's supposed to work? I tried doing `[matcher-combinators.test :refer [match? thrown-match?]]` the other day but the `thrown-match?` refer didn't work.

Reply:
actually, it does work, but only _inside_ `is`, so I think I misremembered: I was trying to refer `thrown-match?` from `matcher-combinators.standalone`, but the only def there is `match?`

Reply:
still, isn't it better to explicitly mention what you're referring to play nice with linters as @philomates previously said?

Reply:
@FelipeCortez if you mean include include them in a `:refer` vector, I have mixed feelings about it. `match?` is not a function, but referring it leads readers of the code to believe it is a function, as you just discovered (it works in `is` because it is a dispatch key for `clojure.test/assert-expr`).

Reply:
@dchelimsky makes sense!